Mathlib/CategoryTheory/Limits/Shapes/Pullback/PullbackCone.lean

```diff
@@ -512,4 +512,79 @@ def PushoutCocone.isoMk {F : WalkingSpan ⥤ C} (t : Cocone F) :
       · dsimp
         simp
 
+section Under
```

Comment:
This should go in `Mathlib/CategoryTheory/ChosenFiniteProducts/Over.lean`

Reply:
Why?

Reply:
Oops wrong file.
Should be `Mathlib/CategoryTheory/Limits/Constructions/Over/Products.lean`.

Reply:
And please keep the comment about why we couldn't just reuse the content of that file.

Reply:
I still don't really see the point of moving this construction to a later file, but okay

Reply:
Then you could also make `Mathlib/CategoryTheory/Limits/Constructions/Over/BinaryProducts.lean` and put it there. The point is `CategoryTheory/Limits/Shapes/*` probably shouldn't mention any specific category.

Reply:
(shouldn't mention specific category for the output `C`; whereas `J` should be specific by nature of what that folder is. and here you're specialising `C`)